.github/workflows/pr.yml

```diff
@@ -5,9 +5,13 @@ env:
   NX_VERBOSE_LOGGING: true
```

Comment:
can we disable the verbose logging again? it's a lot of logs - is it necessary ?

Reply:
For sure!